src/pages/_app.js

```diff
@@ -9,12 +9,19 @@ import { MembersProvider } from '@store';
 import { UserContextProvider } from '@store/user/user-context';
 import { SearchMemberProvider } from '@store/search-members/searchMembers-context';
 import { usePostHog } from 'next-use-posthog';
+import { KeyboardProvider } from '@store/keyboard/context';
 
 const MyApp = (props) => {
   const { Component, pageProps } = props;
   const router = useRouter();
 
   const [loading, setLoading] = useState(false);
+  const [isOptionKey, setIsOptionKey] = useState(false);
+
+  const initialvalue = {
+    isOptionKey,
+    setIsOptionKey,
+  };
```

Comment:
Shouldn't this be in KeyboardContext?

Reply:
I tried it, we will not be able to access it in our root div then, that is why I have put it in here.

Reply:
Which is the root div?